web-marketplace/src/components/organisms/BuyCreditsModal/hooks/useFetchUserBalance.tsx

```diff
@@ -9,12 +9,16 @@ import { UISellOrderInfo } from 'pages/Projects/AllProjects/AllProjects.types';
 
 interface Params {
   selectedSellOrder?: UISellOrderInfo;
+  askDenom?: string;
```

Comment:
what about refactoring this to only take `askDenom` as param?

Reply:
it's only used once, so it would be really a small refactor:

`const userBalance = useFetchUserBalance({ askDenom: selectedSellOrder?.askDenom });`